### DISCUSSION TITLE
Vitesse But only without Vite SSG

### DISCUSSION THREAD

**Discussion by khaledOghli** [antfu/vitesse#463] [Q&A]
What if I just need to remove SSG 
I want the full version of Vitesse but without SSG because I have a lot of errors at the build.

Does anyone know how to make it without any errors?

**Comment by compilekaiten** [comment]
Dude, post any information about the errors.

How is anyone supposed to help you?

**Reply by khaledOghli** [reply]
I'm trying to remove ViteSSG, and I think the error is because now I didn't call the modules/*.ts folder 

`const app = createApp(App)
`
instead of 

```
// export const createApp = ViteSSG(
//   App,
//   { routes, base: import.meta.env.BASE_URL },
//   (ctx) => {
//     // install all modules under `modules/`
//     Object.values(import.meta.glob<{ install: UserModule }>('./modules/*.ts', { eager: true }))
//       .forEach(i => i.install?.(ctx))
//     ctx.app.use(Previewer)
//   },
// )
```

**Reply by khaledOghli** [reply]
```
const app = createApp(App)
Object.values(import.meta.glob<{ install: any }>('./modules/*.ts', { eager: true })).forEach(i => i.install?.(app, router))

```
remove { type UserModule } for SSG for all modules files.

I didn't know if this best way but it's work

**Comment by 1CoolUsername** [comment]
Pretty sure I achieved this in one of the apps Ive built by simply changing the package.json Scripts to a regular Vite build. 
If I recall I had some conflict that would not allow for SSG, I simply changed the build script.
From - "build": "vite-ssg build"
To - "build": "vite build"

Update: I have since revisited this today and corrected the Build back to vite-ssg; the issue I was having with modules is that the window object is not available in ssg on build. I resolved it with the isClient check similar used to the PWA module. There is another discussion or issue were I found the solution mentioned by me in the update.
